boards/v2/others/stm32_min_dev/board.yml

```diff
@@ -0,0 +1,5 @@
+board:
+  name: stm32_min_dev
```

Comment:
this board has not had the _defconfig files updated
```
CONFIG_SOC_SERIES_STM32F1X=y
CONFIG_SOC_STM32F103X8=y
```
And the Kconfig.defconfig file can be deleted, Kconfig.board should be split and renamed with selection disabled

Reply:
Yeah, I fully reviewed this board variant definition scheme